### DISCUSSION TITLE
How refetch data of some specify page when using useInfiniteQuery

### DISCUSSION THREAD

**Discussion by boringplay** [TanStack/query#7054] [Q&A]
In my page, there is an infinite scroll list. When I click on a card in the list, perform some actions, and then return to the list, the data on that card should be updated. How can I refetch the data for a specific page using useInfiniteQuery without having to update all the pages?"

**Comment by TkDodo** [comment]
you can't. logically, all pages form a single cache entry that is just chunked into multiple requests. It's very easy to get into cache inconsistencies if you only update a single page in an infinite query.

if you have too many pages in the cache for a refetch, consider the `maxPages` option.
